app/assets/javascripts/models/data/collection.js.coffee

```diff
@@ -299,4 +299,7 @@ ns.Collection = do (ko
       this[prop] ?= ko.observable(undefined)
       this[prop](jsonObj[prop] ? this[prop]())
 
+    has_feature: (namespace, feature_key) ->
+      return @tags()?[namespace + ".features." + feature_key]?.data && @searchable_attributes
```

Comment:
This should use the existing method to read tags, which lets us be more robust against future tag API changes.

Also "@searchable_attributes" should not be in this method. What if we add another feature that has nothing to do with additional attributes?

So,
    
    has_feature: (key) ->
      @getValueForTag("features.#{key}")